src/main/java/recipeio/recipe/RecipeList.java

```diff
@@ -41,27 +43,38 @@ public Recipe get(int index) {
         return recipes.get(index);
     }
 
+    /**
+     * Used to add recipes from user inputs
```

Comment:
Hi xiao kang, do check the convention of Java Doc: the first line should start with a verb in present tense.

e.g. Adds recipes from user input